spec/features/piece/show_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe "piece show page" do
+  it "should have the name of the piece and composer (dummy data test)" do
```

Comment:
With this dummy data test, are we going to refactor this test when we come back with actual data from the API? Just clarifying with you